pkg/collector/corechecks/containers/docker.go

```diff
@@ -232,7 +232,7 @@ func (d *DockerCheck) Run() error {
 		if c.Network != nil {
 			for _, netStat := range c.Network {
 				if netStat.NetworkName == "" {
-					log.Debugf("Ignore network stat with empty name for container %s: %s", c.ID[:12], netStat)
+					log.Debugf("Ignore network stat with empty name for container %s", c.ID[:12])
```

Comment:
what happened to netStat?

Reply:
`netStat` come from https://github.com/DataDog/datadog-agent/blob/master/pkg/util/containers/metrics/types.go#L9. So an `InterfaceNetStats` without a name is not very interesting to log. We could use `%v` to log the number of bytes and packets sent/received but I don't think it would be very useful.

Container team might have more information on this.